packages/taquito-utils/src/taquito-utils.ts

```diff
@@ -99,6 +99,20 @@ export function b58decode(payload: string) {
   }
 }
 
+/**
+ *
+ * @description b58 decode a string without predfined prefix
+ * @param value
+ * @returns string of bytes
+ */
+// b58decodeRollup or b58decodePrefixless
+export function b58decodeRollup (payload: string) {
```

Comment:
Could the name be confusing between tz4 and txr1, should L2Address be part of the name of the function?